lib/folio_record.rb

```diff
@@ -49,6 +49,8 @@ def sirsi_holdings
       holding = holdings.find { |holding| holding['id'] == item['holdingsRecordId'] }
       next unless holding
 
+      gov_docs_classification = item.dig('location', 'effectiveLocation', 'details', 'searchworksGovDocsClassification')
```

Comment:
I'm surprised we have to snake this data through the `SirsiHolding` at all. It seems to me we could just pluck them out of `record.items` in the traject config.